addons/mail/static/src/components/video_room/video_room.js

```diff
@@ -0,0 +1,100 @@
+odoo.define('mail/static/src/components/video_room/video_room.js', function (require) {
+'use strict';
+
+
+const useStore = require('mail/static/src/component_hooks/use_store/use_store.js');
+const useRefs = require('mail/static/src/component_hooks/use_refs/use_refs.js');
+
+const { Component, useState } = owl;
+
+class VideoRoom extends Component {
```

Comment:
Would also use "RtcRoom" naming here. Right now it's just video part but eventually voice and screen sharing will also be included, and all of them are part of the "Real-Time Communication" (RTC) aspect of the thread. Word "room" is fine to illustrate better the concept of a specific place for RTC.